src/components/input/demoTextarea/index.html

```diff
@@ -0,0 +1,22 @@
+<div layout="column" ng-cloak class="md-inline-form">
+
+  <md-content layout-padding>
```

Comment:
The CSS for `layout-padding` changed recently. We are aware of some issues with it, but the quick workaround is to wrap the content in a `<div>`.